Example/Tests/BONTagComplexStylesTestCase.m

```diff
@@ -0,0 +1,406 @@
+//
+//  BONTagComplexStylingTestCase.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/3/16.
+//
```

Comment:
#nth a couple of these files are missing the copyright notice. I care more because of consistency than legality.